wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js

```diff
@@ -104,11 +116,50 @@ class WorkspaceSwitcherPopup extends SwitcherPopup.SwitcherPopup {
             mask = 0
         }
 
-        if (super.show(backward, binding, mask)){
+        if (this.show(backward, binding, mask)){
             modals.push(this);
         }
     }
 
+    show(backward, binding, mask) {
```

Comment:
What about using `super.show()` and then immediately cancelling the animation using `this._initialDelayTimeoutId`? Could this work? If so, we wouldn't need to duplicate all the code.

Reply:
probably calling `_showImmediately()` after show will be a workaround, I just didn't want it to create a timeout for every monitor as we already need to fix the extra ones that are showing up in the warning logs. I will try the `_showImmediately()` option as it seems to be the cleanest.